process/peer/process.go

```diff
@@ -165,6 +171,11 @@ func (p *validatorStatistics) UpdatePeerState(header data.HeaderHandler) ([]byte
 		return nil, err
 	}
 
+	newRh, _ := p.peerAdapter.RootHash()
```

Comment:
This is code I added for debugging what goes wrong when we see a mismatch on the validator statistics root hash. It should be discussed if we should leave it here - it could be helpful later also

Reply:
this part was removed.